src/store/actions/set_active_period_and_data.js

```diff
@@ -28,14 +29,26 @@ export default firestoreAction(
       .where('parent', '==', parentRef)
       .orderBy('name');
 
+    const objectiveContributorsRef = db
+      .collection('objectiveContributors')
+      .where('item', '==', parentRef);
+
     const activeObjectivesList = await objectivesRef
       .get()
       .then((snapshot) => snapshot.docs.map((doc) => doc.ref));
 
-    if (activeObjectivesList.length) {
+    const objectiveContributorsList = await objectiveContributorsRef
+      .get()
+      .then((snapshot) => snapshot.docs.map((doc) => doc.ref));
+
+    if (activeObjectivesList.length || objectiveContributorsList.length) {
       await bindFirestoreRef('objectives', objectivesRef, { maxRefDepth: 1 });
+      await bindFirestoreRef('objectiveContributors', objectiveContributorsRef, {
+        maxRefDepth: 1,
+      });
```

Comment:
Dette fungerer ikke nå på mål med gammeldagse perioder, men det er kanskje bevisst / like greit å droppe støtte for?